package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "grad23-snakesandladders",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Add a start script so we can launch the program easily.
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node index.js"
  },
```